libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/GoogleFragment.java

```diff
@@ -47,10 +43,10 @@
 
     protected static final String SERVICE_TYPE_GOOGLE = "google";
 
-    @Inject
-    protected Dispatcher mDispatcher;
-    @Inject
-    protected SiteStore mSiteStore;
+    @Inject protected Dispatcher mDispatcher;
+    @Inject protected SiteStore mSiteStore;
+
+    @Inject protected LoginAnalyticsListener mAnalyticsListener;
```

Comment:
It's just a bit overly verbose to have an extra line for each, so we've been following this pattern in FluxC code for some time. Haven't gotten around to writing a checkstyle rule for this yet, so it's not enforced yet.

`@Override` is different because it's used for method declarations and we've conventionally kept those on a separate line for a long while (of course, this is fixable by converting everything to Kotlin, which takes an inlined `override` modifier).

Generally, if I were to write a checkstyle rule, it would probably be "enforce inline annotations for fields, unless those annotations take arguments, and not when that would make the line length of the field declaration exceed 120 characters".